onnxscript/values.py

```diff
@@ -271,12 +299,93 @@ def __init__(
         self.source = source
         self.kwargs = kwargs
         self._param_schemas: Optional[tuple[ParamSchema, ...]] = None
+        self._opschema: Optional[onnx.defs.OpSchema] = None
 
     @property
     def name(self):
         """Returns the function name."""
         return self.opname
 
+    @property
+    def opschema(self) -> Optional[onnx.defs.OpSchema]:
```

Comment:
Is this being used anywhere yet?

Reply:
No. It will be used by the exporter. For now I added tests to make sure all torch_lib functions have opschemas defined

Reply:
Sorry the tests were left out for some reason. I added them back